Source/Scene/Camera.js

```diff
@@ -2853,18 +2853,24 @@ define([
         return Math.max(right, top) * 1.50;
     }
 
-    var MINIMUM_ZOOM = 100.0;
-
     function adjustBoundingSphereOffset(camera, boundingSphere, offset) {
         if (!defined(offset)) {
             offset = HeadingPitchRange.clone(Camera.DEFAULT_OFFSET);
         }
 
+        var DEFAULT_ZOOM = 100.0;
+        var MINIMUM_ZOOM = camera._scene.screenSpaceCameraController.minimumZoomDistance;
+        var MAXIMUM_ZOOM = camera._scene.screenSpaceCameraController.maximumZoomDistance;
         var range = offset.range;
         if (!defined(range) || range === 0.0) {
             var radius = boundingSphere.radius;
-            if (radius === 0.0) {
+            // If the minimumZoomDistance hasn't been set yet, it is equal to 1,
+            if (radius < MINIMUM_ZOOM && MINIMUM_ZOOM > 1) {
```

Comment:
I was taking the approach that if the radius is 0, it would go to the default zoom unless the minimum zoom had been set already. If you zoom all the way to 1, it doesn't look very good, just a bunch of grey, because it is too close to the surface.